.github/workflows/source_clear_cron.yml

```diff
@@ -0,0 +1,16 @@
+name: Source clear
+
+on:
```

Comment:
can we put master only

Reply:
GitHub Actions only runs scheduled jobs on the default branch of your repository (usually master or main)